doc/book/writer.md

```diff
@@ -210,7 +210,7 @@ Method | Description
 `setTitle()` | Set the title of the feed.
 `setDescription()` | Set the text description of the feed.
 `setLink()` | Set a URI to the HTML website containing the same or similar information as this feed (i.e. if the feed is from a blog, it should provide the blog's URI where the HTML version of the entries can be read).
-`setFeedLinks()` | Add a link to an XML feed, whether it is to the feed being generated, or an alternate URI pointing to the same feed but in a different format. At a minimum, it is recommended to include a link to the feed being generated so it has an identifiable final URI allowing a client to track its location changes without necessitating constant redirects. The parameter is an array of arrays, where each sub-array contains the keys "type" and "uri". The type should be one of "atom", "rss", or "rdf".
```

Comment:
@elazar 
Can you also check the [table for `Zend\Feed\Writer\Entry`](https://github.com/zendframework/zend-feed/blob/master/doc/book/writer.md#setting-entry-data-points). There is the same wrong method included. Thanks!